app/src/main/java/com/codepath/apps/critterfinder/network/PetFinderHttpClient.java

```diff
@@ -38,6 +40,22 @@ public void findPetList(JsonHttpResponseHandler handler) {
         client.get(apiUrl, params, handler);
     }
 
+    /**
+     * Return a list of breeds for a particular species
+     * @param species
+     * @param handler
+     */
+    public void getBreedList(SearchFilter.Species species, TextHttpResponseHandler handler) {
+        String apiUrl = getApiUrl("breed.list");
+
+        RequestParams params = new RequestParams();
+        params.put("format", "json");
+        params.put("animal", species.toString());
```

Comment:
@scottrichards you might find this trick useful when wiring up search. I've added toString methods on some of the enums in a search filter. The string value of the enum can match the value pet finder expects. 

For instance:
species.toString()

returns dog or cat.

gender.toString() returns M or F 

useful for quickly converting enums to string values PetFinder wants